solo-chains/docs/bridge/etbereum-to-solo-chain.md

```diff
@@ -0,0 +1,116 @@
+# Ethereum State Verification Diagram
+
+## Altair Sync protocol
+
+### Preliminaries
+- Ethereum epochs consis of 32 slots, where each slot is 12s
+- Ethereum distributes all validators in attestation 32 attestation committees (one per slot)
+- At the beginning of each epoch, each validator is assigned a slot
+- Ethereum pseudo-randomly selects, among all validators, a block producer for a slot n
+- The block produced for slot n needs to be attested by the attestation committee for slot n
+
+
+### The need for a sync protocol
+- For a light client, determining the canonical chain can become expensive (as we need to verify a bunch of attestions)
+- In order to verify a block header, you need the information of the previous header
+
+### The Altair sync protocol
+- A sync committe made up of 512 pseudo-randomly chosen validators is selected every 256 epochs (approx, 27hours)
+- Each block header includes the aggregated signature of the validators in the sync committee,
+- Each header lists the current and the next sync committee
+- Sync committee members are rewarded with 0.1ETH for their service, and they are charged that amount if they dont sign
+- 2/3rds quorum needs to be reached
+- For now sync committee members are not slashed if the vote for a malicious header (although subject to change after EIP-7657)
+
+### Verification PseudoCode
```

Comment:
I would say better to link to the original source as there are bunch of functions that this function calls and few other functions that calls this function in that file and will give anybody reading a complete idea about the algorithm.

Reply:
sounds good, I will add also a summary on what the funtion does

Reply:
added a high level description and linked the pseudocode